store/iavl/store.go

```diff
@@ -147,6 +147,15 @@ func (st *Store) LastCommitID() types.CommitID {
 	}
 }
 
+// PausePruning implements CommitKVStore interface.
+func (st *Store) PausePruning(pause bool) {
```

Comment:
Please add  the `PausablePruner`  interface to the type checks on top of the file so that we have type safety guaranteed.
```go
var(
...
	_ types.PausablePruner = (*Store)(nil)
```